src/SFA.DAS.FAT.Domain/Courses/Ksb.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using SFA.DAS.FAT.Domain.Courses.Api.Responses;
+
+namespace SFA.DAS.FAT.Domain.Courses;
+
+public class Ksb
```

Comment:
Just keep this one as it is same as KsbResponse

Reply:
ok

Reply:
dont understand this comment